components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -205,6 +214,17 @@ class PromptFeature private constructor(
                 }
         }
 
+        @Suppress("MagicNumber")
+        dismissPromptScope = store.flowScoped { flow ->
```

Comment:
I'd also add a comment here why were doing this as it's not obvious. Basically to dismiss popups if a new page page loads while the popup displays, or maybe link to the issue...